examples/website/3d-tiles/app.js

```diff
@@ -54,7 +54,7 @@ export class App extends PureComponent {
         // Update deck.gl viewState, moving the camera to the new tileset
         longitude: cartographicCenter[0],
         latitude: cartographicCenter[1],
-        zoom: zoom + 1.5, // TODO - remove adjustment when Tileset3D calculates correct zoom
+        zoom,
```

Comment:
Not too zoomed out?

Reply:
I think it is better to zoom out a little on the website, users can understand the point cloud better. In high zoom, the model is kind of too sparse, people may not notice the full picture.